process/peer/process.go

```diff
@@ -131,22 +144,37 @@ func (p *validatorStatistics) UpdatePeerState(header data.HeaderHandler) ([]byte
 		return nil, err
 	}
 
-	//previousHeader, err := p.getMetaHeaderFromStorage(header.GetPrevHash())
+	// TODO: Currently there is a bug that shards build a different meta genesis block.
```

Comment:
this bug apparently was only on one branch, so maybe remove the mention of the bug and either leave the todo or fix it.

Reply:
Todo was changed, should be removed when we have the genesis block into the storage